blogs/parameterNames.md

```diff
@@ -0,0 +1,105 @@
+C# 4 introduced named and optional parameters: the ability to set default values for arguments and explicitly label arguments by name instead of just position. Despite being a relatively minor features, named and optional parameters actually do quite a lot to make coding simpler to write, read, and consume. So much so, that I find myself sorely missing this feature when working in languages without it (aka Javascript).
```

Comment:
Javascript has it...kinda.

```
function baz(config) { 
    var foo, bar;
    if (arguments.length === 1) {
        foo = config.foo;
        bar = config.bar;
    } else {
        foo = arguments[0];
        bar = arguments[1];
    }
}
baz({foo: "foo", bar: "bar"});
baz("foo", "bar");
```

Truthfully, I've never used the arguments construct like this.  But I do find myself taking in config objects rather than multiple parameters a lot in javascript.